src/module.c

```diff
@@ -3871,6 +3872,17 @@ int VM_GetClientInfoById(void *ci, uint64_t id) {
     return modulePopulateClientInfoStructure(ci, client, structver);
 }
 
+/*
+Same as ValkeyModule_GetClientInfoById, except it's always for the current client.
+*/
+int VM_GetClientInfo(ValkeyModuleCtx *ctx, void *ci) {
+    if (ci == NULL) return VALKEYMODULE_OK;
+
+    /* Fill the info structure if passed. */
+    uint64_t structver = ((uint64_t *)ci)[0];
+    return modulePopulateClientInfoStructure(ci, ctx->client, structver);
```

Comment:
`VM_GetClientId` also handles the case `ctx->client == NULL`. We should do it here too.